.travis.yml

```diff
@@ -22,15 +24,19 @@ script:
   - gulp build
   - npm test
 
+after_success:
+  - chmod +x push-build.sh
+  - ./push-build.sh
+
 env:
   global:
-    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up installation of html-proofer
+    # speeds up installation of html-proofer
+    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
     - NODE_ENV=production
 
 # Only build pushes to the master branch, PRs, and branches beginning with `test-`.
 branches:
   only:
-    - master
-    - /^test-.*$/
+    - /.*/
```

Comment:
My thinking was that users who push to the repo might want to preview their build without opening a PR. Now that we won't be building PRs we'll need this.

Good point about the section and thanks for catching that comment!